doc/source/tutorial/stats/hypothesis_tests.rst

```diff
@@ -18,4 +11,19 @@ more detailed examples, see the following sections.
 .. toctree::
```

Comment:
Does the toctree seem reasonable to you? I made it in alphabetical order. Also we don't see the documents in the left sidebar, just the main Hypothesis testing section - is that ok?

Reply:
Yeah I was going to mention that this looks much better than before.

<img width="1105" alt="image" src="https://github.com/scipy/scipy/assets/6570539/0b70dbb4-880c-4fb4-8603-f989f41dece1">

<img width="1121" alt="image" src="https://github.com/scipy/scipy/assets/6570539/fa6d1f8a-c167-46be-95c1-14733f1585a4">

I don't think they should show up separately in the sidebar. I think these look fine in the screenshots.